### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by tibetsprague**

### CODE REVIEW

File: webpack.dev.config.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ module.exports = merge(baseConfig, {
     new webpack.NamedModulesPlugin(),
 
     new webpack.EnvironmentPlugin({
-      NETWORK: process.env.NETWORK,
+      NETWORK: "ganache",
```
Comment (dkent600) [inline webpack.dev.config.js]: Does this not break local builds against staging, etc, that are dev builds against non-ganache networks?

File: webpack.prod.config.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ plugins: [
     }),
 
     new webpack.EnvironmentPlugin({
-      NETWORK: process.env.NETWORK,
+      NETWORK: "main",
```
Comment (dkent600) [inline webpack.prod.config.js]: What if want to run a prod build against another network?  Can't do it with this change.

File: webpack.dev.config.js (hunk 1 above)
Comment (tibetsprague) [reply webpack.dev.config.js]: no, that is the default value if process.env.NETWORK is not set, this is how you use EnvironmentPlugin

File: webpack.prod.config.js (hunk 2 above)
Comment (tibetsprague) [reply webpack.prod.config.js]: that is the default value if process.env.NETWORK is not set, this is how you use EnvironmentPlugin

**Pull request merged**